src/blocks/logos/components/logos.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { chunk, flatten } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+const { Component, Fragment } = wp.element;
+const { ResizableBox } = wp.components;
+
+class Logos extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.state = {
+			selectedImage: null,
+		};
+	}
+
+	componentDidUpdate( prevProps ) {
+		if ( ! this.props.isSelected && prevProps.isSelected ) {
+			this.setState( {
+				selectedImage: null,
+			} );
+		}
+	}
+
+	render() {
+		let imageChunks = chunk( this.props.images, 4 );
+
+		return (
+			<Fragment>
+			{ Object.keys( imageChunks ).map( keyOuter => {
+
+				var images = imageChunks[ keyOuter ];
+
+				return (
+					<div className="wrapper" key={ "wrapper-" + keyOuter }>
+						{ images.map( ( img, index ) => {
+							return (
+								<ResizableBox
+									key={ img.id + '-' + keyOuter + '-' + index }
+									className={ classnames(
+										'resize',
+										{
+											'is-selected': ( img.id === this.state.selectedImage ),
+										}
+									) }
+									size={ {
+										width: img.width ? img.width : ( ( 100 / images.length ) + '%' ),
```

Comment:
I think we need to keep the context of how many logos are in the row so we can distribute the 100% between them. I believe this is why we have a little bit of a jump glitch going on when resizing. We can set all of the logos to >100% width right now and when attempting to resize another the resize control jumps oddly.

I think this is going to be a bit complex to solve because we don't always want to distribute the available width between each logo. We do want to make sure the total width never exceed 100% though. I see a few options here.

1. We distribute the width equally on upload and if the width is different than the initial value, we maintain that selected width when resizing other logos. (complex solution)
2. By ensuring we don't exceed 100% a user would have to make one logo smaller before being able to make another larger. (simple solution, but complicated for the user)
3. Disable the first logo's left resize control and the last logo's right resize control. Link the resizing of the logo to the adjacent logo so that moving one control will resize the logos on either side of it. (complex solution, better UX)

Thinking out loud a bit here. I'd love to know both of your thoughts here. // @richtabor @EvanHerman